src/utils/domain.ts

```diff
@@ -56,6 +56,9 @@ export const getAvatarPathWithFallback = (avatar?: string, name?: string) => {
   return avatar ? getAvatarPath(avatar) : placeholder;
 };
 
+export const getInitialFromName = (name: string) =>
```

Comment:
when the name is two words, the initials should be the first letters of each word.